src/integration/pages/homePage/canonicalTests.js

```diff
@@ -37,4 +40,18 @@ export default ({ service, pageData, displayAds }) => {
       });
     }
   });
+
+  describe('VJ Embed', () => {
+    const vjEmbedComponent = document.getElementById('VJ-Embed');
```

Comment:
Are we sure that's the correct ID of the embed component? If not, we might need to add a data-testid so that we can determine if it's actually rendered on the page or not. Let's add a hindi homepage test first, to see if this passes/fails, and then we can figure out what we might need to do to fix it.

Reply:
I have added a data-testid of 'embed' and a hindi homepage test. It passes.